### DISCUSSION TITLE
Help us name the 1.9 release 🎃

### DISCUSSION THREAD

**Discussion by cmpadden** [dagster-io/dagster#25341] [Community]
![image](https://github.com/user-attachments/assets/10c7103c-cd96-426e-9d30-13595aac3048)

Hey folks! @cmpadden  and @C00ldudeNoonan  from the Dagster team here.

Dagster v1.9 is just around the corner and want your help to name it!

Every major and minor release gets named after a hit single or an album from a pop diva. Sometimes, the name of the song/album is inspired by what changes are coming with the release. But we'll also consider something _spooky_ since we're targeting to release 1.9 on Halloween! 🎃 

For inspiration, 1.9 will have:

- General availability of declarative automation
- BI Integrations like Looker, Tableau, Sigma, and Power BI
- Airlift for observing and migration Airflow projects
- Improved asset metadata collection for our ETL integrations
- And more, so stay tuned!

## Previous names

Here are some of the songs and albums we've previously used:

- [Call Me Maybe](https://www.youtube.com/watch?v=fWNaR-rxAic) (Carly Rae Jepsen)
- [How Will I Know?](https://www.youtube.com/watch?v=m3-hY-hlhBg) (Whitney Houston)
- [Material Girl](https://www.youtube.com/watch?v=xeddEMuM0dA) (Madonna)
- [Smooth Operator](https://www.youtube.com/watch?v=4TYv2PhG89A) (Sade)
- [Formation](https://www.youtube.com/watch?v=WDZJPJV__bQ) (Beyonce)
- [Thank U, Next](https://www.youtube.com/watch?v=gl1aHhXnN1k) (Ariana Grande)
- [Hello](https://www.youtube.com/watch?v=YQHsXMglC9A) (Adele)
- [Cool for the Summer](https://www.youtube.com/watch?v=il9nqWw9W3Y) (Demi Lovato)
- [Lucky Star](https://www.youtube.com/watch?v=ThHz9wlBeLU) (Madonna)
- [The Edge of Glory](https://www.youtube.com/watch?v=QeWBS0JBNzQ) (Lady Gaga)
- [Laundry Service](https://www.youtube.com/watch?v=7BPE5sR6oUU&list=PL035E8452800ACEA5) (Shakira)
- [In the Zone](https://www.youtube.com/watch?v=we8h_2RFAyA) (Britney Spears)
- [Waiting to Exhale](https://www.youtube.com/watch?v=_FT7ciAPuJk&list=PL_sN_rrP4od0gZ4C3LGkgcaRpVa_koQX4) (Whitney Houston, et al.)
- [Love Plus One](https://www.youtube.com/watch?v=5_msHpEa3_Y) (Haircut 100)

🎵 _[Listen to the full playlist](https://open.spotify.com/playlist/476UFttwzU8476ki1Aqcdd)_ 🎵 

Drop a comment with your suggestions, and upvote the suggestions you like!

**Comment by stevenmurphy12** [comment]
Dusty Springfield - Spooky - https://www.youtube.com/watch?v=h8bAbvk5Hs4

**Reply by PedramNavid** [reply]
this one is on REPEAT lately for me

**Reply by cmpadden** [reply]
We're going with Spooky! Thanks for your suggestion, @stevenmurphy12.

**Comment by sryza** [comment]
Thriller
(cred to @salazarm )

**Comment by sryza** [comment]
Monster Mash

**Reply by nigelainscoe** [reply]
Here's the link for those who may not know it https://www.youtube.com/watch?v=tsPdVsIXqzU

**Comment by bahill** [comment]
Running Up That Hill https://www.youtube.com/watch?v=bV0RAcuG2Ao

**Comment by EvanZ** [comment]
Graveyard (Halsey)

**Comment by akan72** [comment]
Astro Zombie (The Misfits)

**Comment by aksestok** [comment]
[Schedules (Sigrid)](https://youtu.be/RrRDEleMo5g?si=fNI4W_Zwa-iDD_9g)

**Comment by neuromantik33** [comment]
West Coast by Lana Del Rey.. you guys are based in SF no? :wink:
